<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OgmToolUI</class>
 <widget class="QWidget" name="OgmToolUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="statusTip">
      <string>widget-miniTop</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="Tool">
        <property name="statusTip">
         <string>lbl-f18-title</string>
        </property>
        <property name="text">
         <string>Tool</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>889</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>30</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>210</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>210</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="3,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_11" native="true">
              <property name="statusTip">
               <string>widget-tool-top</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7" stretch="4,1">
               <item>
                <widget class="QLabel" name="lblToolUdx">
                 <property name="statusTip">
                  <string>lbl-tool-icon</string>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblToolUdxName">
                 <property name="statusTip">
                  <string>lbl-tool-name</string>
                 </property>
                 <property name="text">
                  <string>UDX Data</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_12" native="true">
              <property name="statusTip">
               <string>widget-tool-bottom</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QToolButton" name="btnToolUdxDataRun">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="statusTip">
                  <string>btn-tool</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btnToolUdxInfo">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="statusTip">
                  <string>btn-tool</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>210</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>210</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="3,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_15" native="true">
              <property name="statusTip">
               <string>widget-tool-top</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9" stretch="4,1">
               <item>
                <widget class="QLabel" name="lblToolA">
                 <property name="statusTip">
                  <string>lbl-tool-icon</string>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="statusTip">
                  <string>lbl-tool-name</string>
                 </property>
                 <property name="text">
                  <string>UDX Schema</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_16" native="true">
              <property name="statusTip">
               <string>widget-tool-bottom</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QToolButton" name="btnToolUdxSchemaRun">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="statusTip">
                  <string>btn-tool</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btnToolAInfo">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="statusTip">
                  <string>btn-tool</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>210</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>210</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="3,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_13" native="true">
              <property name="statusTip">
               <string>widget-tool-top</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8" stretch="4,1">
               <item>
                <widget class="QLabel" name="lblToolOms">
                 <property name="statusTip">
                  <string>lbl-tool-icon</string>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="statusTip">
                  <string>lbl-tool-name</string>
                 </property>
                 <property name="text">
                  <string>OMS Packing</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_14" native="true">
              <property name="statusTip">
               <string>widget-tool-bottom</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QToolButton" name="btnToolOmsRun">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="statusTip">
                  <string>btn-tool</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btnToolOmsInfo">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="statusTip">
                  <string>btn-tool</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_8" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>210</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>210</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_17" native="true">
              <property name="statusTip">
               <string>widget-tool-top</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10" stretch="4,1">
               <item>
                <widget class="QLabel" name="lblToolB">
                 <property name="statusTip">
                  <string>lbl-tool-icon</string>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="statusTip">
                  <string>lbl-tool-name</string>
                 </property>
                 <property name="text">
                  <string>Model Server Manager</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_18" native="true">
              <property name="statusTip">
               <string>widget-tool-bottom</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QToolButton" name="btnToolBRun">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>manage your default server</string>
                 </property>
                 <property name="statusTip">
                  <string>btn-tool</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btnToolBInfo">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="statusTip">
                  <string>btn-tool</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_9" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>210</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>210</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11" stretch="3,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_19" native="true">
              <property name="statusTip">
               <string>widget-tool-top</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_12" stretch="4,1">
               <item>
                <widget class="QLabel" name="lblToolData">
                 <property name="statusTip">
                  <string>lbl-tool-icon</string>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="statusTip">
                  <string>lbl-tool-name</string>
                 </property>
                 <property name="text">
                  <string>Data Server Manager</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_20" native="true">
              <property name="statusTip">
               <string>widget-tool-bottom</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QToolButton" name="btnToolDataRun">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="statusTip">
                  <string>btn-tool</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btnToolDataDesc">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="statusTip">
                  <string>btn-tool</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_10" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
